providers/common/capabilities.c

```diff
@@ -135,23 +145,28 @@ static const OSSL_PARAM param_group_list[][10] = {
     TLS_GROUP_ENTRY("secp192k1", "secp192k1", "EC", 17),
 #  endif
     TLS_GROUP_ENTRY("secp192r1", "prime192v1", "EC", 18),
+    TLS_GROUP_ENTRY("P-192", "prime192v1", "EC", 18), /* Alias of above */
 #  ifndef FIPS_MODULE
     TLS_GROUP_ENTRY("secp224k1", "secp224k1", "EC", 19),
 #  endif
     TLS_GROUP_ENTRY("secp224r1", "secp224r1", "EC", 20),
+    TLS_GROUP_ENTRY("P-224", "secp224r1", "EC", 20), /* Alias of above */
 #  ifndef FIPS_MODULE
     TLS_GROUP_ENTRY("secp256k1", "secp256k1", "EC", 21),
 #  endif
     TLS_GROUP_ENTRY("secp256r1", "prime256v1", "EC", 22),
+    TLS_GROUP_ENTRY("P-256", "prime256v1", "EC", 22), /* Alias of above */
     TLS_GROUP_ENTRY("secp384r1", "secp384r1", "EC", 23),
+    TLS_GROUP_ENTRY("P-384", "secp384r1", "EC", 23), /* Alias of above */
     TLS_GROUP_ENTRY("secp521r1", "secp521r1", "EC", 24),
+    TLS_GROUP_ENTRY("P-521", "secp521r1", "EC", 24), /* Alias of above */
 #  ifndef FIPS_MODULE
     TLS_GROUP_ENTRY("brainpoolP256r1", "brainpoolP256r1", "EC", 25),
     TLS_GROUP_ENTRY("brainpoolP384r1", "brainpoolP384r1", "EC", 26),
     TLS_GROUP_ENTRY("brainpoolP512r1", "brainpoolP512r1", "EC", 27),
 #  endif
-    TLS_GROUP_ENTRY("x25519", "x25519", "X25519", 28),
-    TLS_GROUP_ENTRY("x448", "x448", "X448", 29),
+    TLS_GROUP_ENTRY("x25519", "X25519", "X25519", 28),
+    TLS_GROUP_ENTRY("x448", "X448", "X448", 29),
 # endif /* OPENSSL_NO_EC */
```

Comment:
I think the alias changes should be moved to a separate PR.

Reply:
I mean all the alias-related changes from this file.

Reply:
Without them things will break after this PR. You'll no longer be able to specify NIST names for groups, which you can prior to this PR. Therefore I think they should stay here.